config/locales/en.yml

```diff
@@ -66,7 +66,6 @@ en:
         no_address_found: "No addresses found matching those details"
     date_of_birth: "What is your date of birth?"
     teacher_reference_number: "What is your teacher reference number (TRN)?"
-    national_insurance_number: "What's your National Insurance number?"
```

Comment:
Is this unused then?

Reply:
It's a duplicate, there's a 2nd value `national_insurance_number: "What is your National Insurance number?"`